gap/perm.gi

```diff
@@ -361,12 +363,15 @@ end;
 FindHomMethodsPerm.ThrowAwayFixedPoints :=
   function( ri, G )
       # Check, whether we can throw away fixed points
-      local gens,hom,l,n,o;
+      local gens,hom,l,n,largest,o;
 
       gens := GeneratorsOfGroup(G);
       l := List(gens,StoredPointsPerm);
       n := NrMovedPoints(G);
-      if 2*n > Maximum(l) or 3*n > LargestMovedPoint(G) then  # we do nothing
+      largest := LargestMovedPoint(G);
+      if (2*n > Maximum(l) or 3*n > largest)
+              and not (IsPrimitive(G) and largest > n)
```

Comment:
It's just too expensive to check `IsPrimitive`.

Instead we should either...
1. enhance the  large base group to also check for fixed points and throw them away (either with its own code or by forcing a `FindHomMethodsPerm.ThrowAwayFixedPoints` step with this limit removed.

2. improve the large base group case to not require that `MovedPoints(G) = [1..LargestMovedPoints(G)]`

Reply:
@ssiccha any plans to revisit this?